src/org/opendatakit/briefcase/export/ExportConfiguration.java

```diff
@@ -149,12 +161,25 @@ public ExportConfiguration setExportDir(Path path) {
   }
 
   public ExportConfiguration setPemFile(Path path) {
-    this.pemFile = Optional.ofNullable(path);
+    //Check if path is a valid pem file with privateKey
+    if (PrivateKeyUtils.isValidPrivateKey(path)) {
```

Comment:
This is a wrong use of Optionals. You could write this instead: 

```java
public ExportConfiguration setPemFile(Path path) {
  this.pemFile = Optional.ofNullable(path).filter(PrivateKeyUtils::isValidPrivateKey)
}
```

If you do that, you don't need `removePemFile()` any more.

Reply:
wow will change it.